modules/sdk/src/main/scala/org/tessellation/sdk/domain/trust/storage/OrdinalTrustMap.scala

```diff
@@ -0,0 +1,34 @@
+package org.tessellation.sdk.domain.trust.storage
+
+import org.tessellation.schema.SnapshotOrdinal
+
+import derevo.cats.eqv
+import derevo.circe.magnolia.encoder
+import derevo.derive
+import eu.timepit.refined.auto._
+import eu.timepit.refined.types.numeric.NonNegLong
+
+@derive(eqv, encoder)
+case class OrdinalTrustMap(
+  ordinal: SnapshotOrdinal,
+  peerLabels: PublicTrustMap,
+  trust: TrustMap
+)
+
+object OrdinalTrustMap {
+
+  val ordinalTrustUpdateInterval: NonNegLong = 1000L
+
+  val ordinalTrustUpdateDelay: NonNegLong = 500L
+
+  def empty: OrdinalTrustMap = OrdinalTrustMap(SnapshotOrdinal.MinValue, PublicTrustMap.empty, TrustMap.empty)
+
+  def initial: OrdinalTrustMap = OrdinalTrustMap(SnapshotOrdinal(ordinalTrustUpdateInterval), PublicTrustMap.empty, TrustMap.empty)
+
+  def advanceInterval(trust: OrdinalTrustMap): OrdinalTrustMap = OrdinalTrustMap(
+    trust.ordinal.plus(ordinalTrustUpdateInterval),
+    PublicTrustMap.empty,
+    TrustMap.empty
+  )
```

Comment:
I still have an issue with this function. Why can't you advance the ordinal and create a new trust map? it's misleading as you basically wipe out all the trust in here

Reply:
This is only used to advance the `next` `OrdinalTrustMap`. When advancing `next`, the `PublicTrustMap` and `TrustMap` are emptied and the ordinal is advanced to the next time that `next` will be updated. To address your concerns, how about re-naming this function `resetAndAdvance`?

Reply:
I think there is nothing bad about the name but you can expose the purpose in the schema that you update these two things separately

Reply:
Re-factored the function to:

```
  def advanceInterval(
    trust: OrdinalTrustMap,
    publicTrustMap: Option[PublicTrustMap] = none,
    trustMap: Option[TrustMap] = none
  ): OrdinalTrustMap = OrdinalTrustMap(
    trust.ordinal.plus(ordinalTrustUpdateInterval),
    publicTrustMap.getOrElse(PublicTrustMap.empty),
    trustMap.getOrElse(TrustMap.empty)
  )
```

Reply:
In these contexts, there are no differences between `Map.empty` and `None`. @ryle-goehausen has also asked that the `updateNext` functions do not update if their source maps are empty. I've reverted this function.

Reply:
Due to re-factors for this comment

https://github.com/Constellation-Labs/tessellation/pull/684#discussion_r1273413831

this function has been removed.